contracts/governance/extensions/GovernorWithERC20Votes.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../IGovernor.sol";
+import "../../token/ERC20/extensions/ERC20Votes.sol";
+
+/**
+ * @dev Extension of {Governor} for voting weight extraction from a Comp or {ERC20Votes} token.
+ *
+ * _Available since v4.3._
+ */
+abstract contract GovernorWithERC20Votes is IGovernor {
+    ERC20Votes public immutable token;
+
+    constructor(address token_) {
```

Comment:
Now that this contract is not COMP-compatible I definitely think we should add a `quorum` argument. We basically need to express a fraction of the past total supply. We had discussed that using percentages was not necessarily sufficient if someone wants to use a fractional percentage (e.g. 30.5%). We also considered parts per thousand. We could also use 18 decimals just like ETH and most token balances.

I don't know if any of this is worth it. I just had a look at several projects and they all use relatively large numbers, above 25%, except Uniswap at 4%.

It may be better for us to stick to percentages, numbers from 0 to 100, and if someone needs more granularity they can fork this module which is very simple.